### ISSUE TITLE
Indicator if bookmark has been updated

### ISSUE THREAD

**Issue by BasThomas** [rnystrom/GitHawk#683]
From https://github.com/rnystrom/GitHawk/pull/601#issuecomment-338176128

Think this is quite a tricky one, depending on the type of bookmark. What is "updated"?

**Comment by Sherlouk** [comment]
In my head this was exclusive to Issues/PRs, if the thread has been updated since you last looked

**Comment by BasThomas** [comment]
Yep, that can work. At least as a first step, although I think Issue+PRs will cover pretty much everything huh

**Comment by Sherlouk** [comment]
Marking as low though because for the most part (granted there are exceptions) you'll be subscribed to the post anyway. So you'll receive a notification through typical means, was more of a "In case you missed it" warning I dunno
